packages/react-devtools-shared/src/__tests__/componentStacks-test.js

```diff
@@ -86,4 +86,41 @@ describe('component stack', () => {
       '\n    in Example (at **)',
     );
   });
+
+  // @reactVersion >=16.9
```

Comment:
```suggestion
```

I think its fair to test this behaviour against `react` from main, not against `react` 17 or 18.